packages/api/cms-api/package.json

```diff
@@ -81,6 +82,7 @@
     },
     "devDependencies": {
         "@aws-sdk/types": "^3.47.0",
+        "@azure/storage-blob": "^12.0.0",
```

Comment:
This is already a dependency, shouldn't be a dev dependency

Reply:
af787c5073f8ed3e1a4fcf3d1be9833207774e09